pkg/monitoring/rules/alerts/operator_alerts.go

```diff
@@ -80,4 +88,51 @@ func operatorAlerts() []promv1.Rule {
 			},
 		},
 	}
+
+	if rule, created := createVMOutdatedMachineTypeRule(); created {
```

Comment:
the alert should always exist, and the condition for firing should be in the alert expression